SPECS/ceph/ceph.signatures.json

```diff
@@ -1,5 +1,5 @@
 {
  "Signatures": {
-  "ceph-16.2.5.tar.gz": "b8206916fa7993e0394a0c31a09564b866fd13af6968aaecdea75ae51a6c46de"
+  "ceph-16.2.10.tar.gz": "df8dbe9d45779825fb4097d94a72847e7405089c0999541e62779f2705bf02d5"
```

Comment:
Why 16.2.10 and not 17.2.2? [NIST](https://nvd.nist.gov/vuln/detail/CVE-2022-0670#range-8182707) seems to suggest that's the first version with this CVE patched.

Reply:
For our major version (Pacific, 16.X.X), 16.2.10 is the version with the fix. See here https://docs.ceph.com/en/latest/security/CVE-2022-0670/

Reply:
Amazing, thank you! In that case please also send NIST an e-mail, so they'd update their database to include 16.2.10 as a fixed version. It's the polite thing to do and also without it our tooling will keep complaining about `ceph`.;)

Reply:
email is sent